paddle/fluid/framework/details/op_handle_base.h

```diff
@@ -31,6 +33,17 @@ constexpr char kLocalExecScopeName[] = "@LOCAL_SCOPE@";
 // It's responsible for populating necessary fields of ir::Node.
 class OpHandleBase {
  public:
+  /**
+   * NOTE(zjl): Some op should have higher priority than others.
+   * The higher priority op would run first without switching
+   * threads in FastThreadedSSAGraphExecutor.
+   *
+   * Currently, EagerDeletionOpHandle has the highest priority.
+   * This priority settings speed up gc 15% in Transformer
+   * V100 8-GPU model.
+   */
```

Comment:
line40~44 should be placed to [here](https://github.com/PaddlePaddle/Paddle/pull/16993/files#diff-d9f7d8760a10c9ad6e5e0f8049971939R43).

Reply:
Done.